import sys

print(sys.argv[1])

# raise Exception("Bye!")
